frontend/src/store/alert-store.js

```diff
@@ -14,7 +15,15 @@ const useAlertStore = create((setState) => ({
       const title = details.title;
       const notificationTitle =
         title || (details.type === "error" ? "Failed" : "Success");
-      return { alertDetails: { ...details, title: notificationTitle } };
+      const duration =
+        details.duration || details.type === "error" ? 0 : undefined;
```

Comment:
@jagadeeswaran-zipstack, if the `details.duration` value is `0`, it will be considered `false` and the else part will be executed. We need to check whether `details.duration` is a `number` and whether it is greater than or equal to zero. If this is `true`, the function can proceed with the same value. If `false`, it can use the fallback value depending on other factors.

Please create a function to check if a variable contains a non-negative number and place it inside the `GetStaticData.js` file. Here's a reference for that function. Feel free to modify it as needed.

```
const isPositiveInteger = (value, includeZero = true) => {
  // Check if the value is a finite number and is an integer (no decimal places)
  if (Number.isFinite(value) && Number.isInteger(value)) {

    // Determine the minimum value based on the includeZero parameter
    const minValue = includeZero ? 0 : 1;

    // Check if the value is greater than or equal to the minimum value
    return value >= minValue;
  }

  // If any of the conditions fail, return false
  return false;
};
```

Reply:
Thanks for the catch.